File: Task2.js
	let score1 = 100;

	let score2 = 10;

	let score3 = 40;
	

        let theSum = score1 + score2 + score3;

	let theAverage = theSum / 3.0;
	

       
        if (theAverage >= 90 && theAverage <= 100) {
        console.log("A");

        } else if (theAverage >= 80 && theAverage < 90) {
         console.log("B");

        }
	 else if (theAverage >= 70 && theAverage < 80) {
          console.log("C");

        }
	 else if (theAverage >= 60 && theAverage < 70) {
          console.log("D");

        }
	 else if (theAverage >= 0 && theAverage < 60) {
          console.log("YOU ARE A FAILURE");

        } 
	else {
         console.log("INVALID INPUT");
        }

File: Task2.py
def main():
    score1 = 100
    score2 = 30
    score3 = 40

    the_sum = score1 + score2 + score3

    the_average = the_sum / 3.0  

    if 90 <= the_average <= 100:
        print("A")

    elif 80 <= the_average < 90:
        print("B")

    elif 70 <= the_average < 80:
        print("C")

    elif 60 <= the_average < 70:
        print("D")

    elif 0 <= the_average < 60:
        print("YOU ARE A FAILURE")

    else:
        print("INVALID INPUT")


main()